### DISCUSSION TITLE
تبلیغات بیجا در پنل کاربر

### DISCUSSION THREAD

**Discussion by zekerya** [hiddify/Hiddify-Manager#480] [General سایر]
اول تشکر میکنم از زحماتتون بعد نقد من از پنل کاربرها اینه که تبلیغ زیادی برای هیدیفای. لینک تلگرام ،توییتر و گیتهاب و لوگو . خیلی ضایع بعد تازه به کاربر پیشنهادم میده که پنل رو نصب هم کنه. 
کسی که از ما پروکسی تهیه کنه یا علمشو نداره یا وقتشو یا امکاناتش. دیگه این همه تبلیغ بی مورد در وب کاربر لازم نیست.
همین حرف رو تلگرام زدم پاک کردین. ما داریم با سانسور مبارزه میکنیم مثلا. امیدوارم این مساله رو لحاظ کنید.
باز هم تشکر از وقت و تلاش و زحماتتون .

**Comment by parsatavafi** [comment]
تبلیغات "بیجا"؟ ایشون شب و روزشون رو گذاشتند تا پنل رو برای ما اماده کنند و نمیشه همه چیز پنل خوب باشه و این که محصولشون {رو که به رایگان در اختیار عام قرار دادند} رو معرفی کنند "بیجا" باشه! دوست عزیزم ایشون به عنوان برنامه نویس سیستم حق قراردادن کپی رایت یا سایر مسائل دیگه رو در پنل کاربر و پنل ادمین نمایش بدهند و فکر نمی کنم نشون دادن کپی رایت و اطلاعات پروژه ای که به رایگان در اختیار شما قرار گرفته تبلیغ محسوب بشه و به مبارزه ی شما با سانسور لطمه ای بزنه. در تمام دنیا وقتی شما از نسخه رایگان اسکریپت یا سیستمی استفاده می کنید تا جای ممکن اطلاعات سازنده نمایش داده میشه و برای حذف لوگو یا کپی رایت پلاگین white label جداگانه قرار میدهند که شما هزینه ای بابت اش پرداخت می کنید (چون استفاده ی تجاری دارید و ازش کسب درآمد می کنید و نمیخواهید کاربراتون یاد بگیرند سیستمی مثل سیستم شما راه اندازی کنند) و با فعالسازیش تمام اطلاعات سازنده از پنل کاربری و ادمین حذف میشه. من فکر می کنم  @hiddify1 باید نسبت به ارائه پلاگین white label به صورت تجاری اقدام کنند.
(یه پرانتز هم باز کنم شما با یک فایل CSS خیلی ساده و اضافه کردنش به هدر سایت میتونید اطلاعات سازنده رو نمایش ندید! البته فکر می کنم برنامه نویس حساسیتی نداشتند که اطلاعات تغییر کنه وگرنه خیلی از جاها رو کد می کردند که کسی نتونه دستکاریش کنه)
